utils/Addresses.json

```diff
@@ -309,7 +315,7 @@
         "addr": "0xd5F9BaE72F6D8334ac4BB6C14463A81A43037E1C",
```

Comment:
Confirmed this is the correct ETH fee splitter:
- `SPLIT_TOTAL` is 10,000, `splitA` and `splitB` are both 5,000 each, which is correct
- `metaMorphoVault` has the correct address: [0xa0E430870c4604CcfC7B38Ca7845B1FF653D0ff1](https://basescan.org/address/0xa0E430870c4604CcfC7B38Ca7845B1FF653D0ff1)
- `token` has the correct address of the WETH token on Base: [0x4200000000000000000000000000000000000006](https://basescan.org/address/0x4200000000000000000000000000000000000006)
- `mToken` has the correct address of the mWETH mToken on Base: [0x628ff693426583D9a7FB391E54366292F509D457](https://basescan.org/address/0x628ff693426583D9a7FB391E54366292F509D457)